addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -46,11 +47,12 @@
                             />
                         </div>
                     </div>
-                    <div 
+                    <div
                         t-if="propertiesList.length % props.columns === propertiesListGroup_index"
-                        t-attf-class="o_field_property_add {{props.columns !== 1 ? 'g-col-2' : ''}}">
+                        class="o_field_property_add"
+                        t-att-class="{'g-col-2': props.columns !== 1, 'd-none': state.hideAddButton}">
                         <button
-                            t-if="!props.readonly &amp;&amp; state.canChangeDefinition"
+                            t-if="!props.readonly &amp;&amp; state.canChangeDefinition &amp;&amp; definitionRecordId"
```

Comment:
Not exactly sure about this change. Does not seem directly related to the feature we added but I'm probably missing something.